### PR TITLE
Ready for review

### PR DESCRIPTION
**Pull request by dyachenko-yaroslav**

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+class ApplicationController < ActionController::API
+  include DeviseTokenAuth::Concerns::SetUserByToken
+
+  private
+
+  def root_params
+    params.require(:data).require(:attributes)
+  end
+
+  def ok
```
Comment (AlexTua) [inline app/controllers/application_controller.rb]: move responses to Module and include it

File: app/channels/application_cable/connection.rb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+module ApplicationCable
```
[the remark this answers is not in the document]
Comment (dyachenko-yaroslav) [reply app/channels/application_cable/connection.rb]: done

File: app/channels/application_cable/channel.rb [hunk 3]
```diff
@@ -0,0 +1,4 @@
+module ApplicationCable
```
[the remark this answers is not in the document]
Comment (dyachenko-yaroslav) [reply app/channels/application_cable/channel.rb]: done

File: app/controllers/api/v1/comments_controller.rb [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+        error 422, 'Unprocessable entity'
+        error 401, 'Unauthorized'
+        formats %w[json]
+      end
+
+      def_param_group :comment do
+        param :data, Hash, required: true do
+          param :attributes, Hash, required: true do
+            param :text, String, required: true
+            param :image, ActionDispatch::Http::UploadedFile
```
[the remark this answers is not in the document]
Comment (dyachenko-yaroslav) [reply app/controllers/api/v1/comments_controller.rb]: done

File: app/controllers/api/v1/comments_controller.rb [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+          end
+        end
+      end
+
+      api :GET, '/v1/tasks/:task_id/comments', "Get all task's comments"
+      def index
+        render json: @comments, status: :ok
```
[the remark this answers is not in the document]
Comment (dyachenko-yaroslav) [reply app/controllers/api/v1/comments_controller.rb]: done

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (dyachenko-yaroslav) [reply app/controllers/application_controller.rb]: done

**Pull request closed without merging**